go.mod

```diff
@@ -2,6 +2,17 @@ module k8s.io/test-infra
 
 replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f
 
+// Pin all k8s.io staging repositories to kubernetes-1.15.3.
+// When bumping Kubernetes dependencies, you should update each of these lines
+// to point to the same kubernetes-1.x.y release branch before running update-deps.sh.
+replace (
```

Comment:
Is there any way we can avoid these terrible replace stanzas?

Reply:
Not really - because Kubernetes doesn't use semver, if we import any other module that depends on one of these dependencies at an older revision, it may select this older revision instead of the one we want to use, and due to lack of semver, it's possible (and in reality, quite likely) that the selected versions may be incompatible with each other.

By setting this here with a replace directive, it guarantees that these module versions will be used for these packages. It's a necessary evil right now with go modules + kubernetes development 😬

Reply:
We *could* get rid of them now, and if we did it *might* build, but whenever we touch the go.mod file we run the risk of breaking the version of any particular k8s dependency that gets selected 🙃 ✨_magic_ ✨

Reply:
FYI this is breaking the ability to `go get` test-infra.